src/ec/packages/ec-component-text-input/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ecl-twig/ec-component-text-input",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Text Input",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@ecl/ec-component-form-feedback-message": "^2.1.0",
```

Comment:
For the sake of consistency, the twig component should only rely on the specs. That's what has been done so far. If we decide to depend on the vanilla implementation (which could make sense), then we have to update the other components as well.

What shall we do?

Reply:
this component is using css classes from vanilla components, so I don't see any reason not to put them in dependencies...
but indeed it could have been done for other components too.